golang/go-cloud-run-hello-world/.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Run on Cloud Run",
+      "type": "cloudcode.kubernetes",
+      "request": "launch",
+      "skaffoldConfig": "${workspaceFolder}/skaffold.yaml",
+      "watch": true,
+      "cleanUp": true,
+      "portForward": true
+    },
+    {
+      "type": "go",
+      "request": "launch",
+      "name": "Launch (local)",
+      "mode": "auto",
+      "program": "${workspaceFolder}/."
+    },
+  ]
+}
```

Comment:
I'm not sure what to do with this file, especially given the discussion to leave skaffold out of the Cloud Run samples.